src/main/groovy/jenkins/automation/builders/JsJobBuilder.groovy

```diff
@@ -4,14 +4,35 @@ import javaposse.jobdsl.dsl.DslFactory
 import javaposse.jobdsl.dsl.Job
 import jenkins.automation.utils.ScmUtils
 
+
+/**
+ * @param name          Job name
+ * @param description   Job description
+ * @param pollSchedule  Optional
```

Comment:
Agreed. Well said. Write it  and it will also show up here: http://cfpb.github.io/jenkins-automation/